### DISCUSSION TITLE
星界链推出了捆绑alas的云手机

### DISCUSSION THREAD

**Discussion by FateInMist** [LmeSzinc/AzurLaneAutoScript#3061] [General]
测试下来速度还挺快的，每个月贵2块钱，比云服务器省多了。只是刚开始配置麻烦点，不能直接复制配置文件了。
INFO     21:16:24.618 │ BENCHMARK RESULTS                                                                                       
                                            ┏━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━┓                                             
                                            ┃ Screenshot    ┃ Time   ┃ Speed      ┃                                             
                                            ┡━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━┩                                             
                                            │ ADB           │ 0.245s │ Fast       │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ ADB_nc        │ 0.125s │ Very Fast  │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ uiautomator2  │ 0.521s │ Slow       │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ aScreenCap    │ 0.070s │ Ultra Fast │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ aScreenCap_nc │ 0.088s │ Ultra Fast │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ DroidCast     │ 0.226s │ Fast       │                                             
                                            ├───────────────┼────────┼────────────┤                                             
                                            │ DroidCast_raw │ 0.036s │ Ultra Fast │                                             
                                            └───────────────┴────────┴────────────┘                                             
INFO     21:16:24.630 │ Recommend screenshot method: DroidCast_raw (0.036s)                                                     
                                               ┏━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┓                                               
                                               ┃ Control      ┃ Time   ┃ Speed  ┃                                               
                                               ┡━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━┩                                               
                                               │ ADB          │ 0.264s │ Slow   │                                               
                                               ├──────────────┼────────┼────────┤                                               
                                               │ uiautomator2 │ 0.147s │ Medium │                                               
                                               ├──────────────┼────────┼────────┤                                               
                                               │ minitouch    │ 0.050s │ Fast   │                                               
                                               └──────────────┴────────┴────────┘                                               
INFO     21:16:24.637 │ Recommend control method: minitouch (0.050s)                                                            
INFO     21:16:24.639 │ [alas] exited. Reason: Finish 

星界链官方：怎么都在玩碧蓝航线

**Comment by guoh064** [comment]
用了一段时间之后怎么样，有截图变慢的现象吗？

**Reply by FateInMist** [reply]
刚用一天，还不好说。之前的普通版连着挂了一个月才发现截图都往5秒去了。

**Reply by guoh064** [reply]
我用的38元一个月的，照样卡
